docs/controlling-fixtures/changing-fixture-attributes.md

```diff
@@ -85,16 +87,25 @@ Some other things to know about attributes:
     mastering the RGB levels when the fixture itself does not provide an
     intensity channel.
 
+## Intensity wheel (Diamond 9 only)
 
+On the Diamond 9 a dedicated wheel is provided to set the intensity of selected
+fixtures. A display above the wheel shows the wheel display for intensity. This wheel always controls
+intensity and is not changed by the Attribute Bank buttons.
```

Comment:
I don't think that is described anywhere in the manual - how does that work? (and for the trackball)

Reply:
My bad, it's not done using the the level wheel `@` key (this works like the other wheel `@` keys), it's done using `<Assign>`.

So, hitting `<Assign>` from the root menu takes you to the "Track Ball Assign" menu (which I've just noticed spuriously contains a rogue "Level Wheel" assignment softkey in 14.0 release.... 👀🐛 )

It looks like this:
![image](https://user-images.githubusercontent.com/7331231/105902700-e6a43400-6016-11eb-8efc-ccb718e40b9d.png)

The softkeys allow you to select an axis to affect, and then hitting the wheel `@` key for an attribute wheel currently showing a given parameter binds that axis to that parameter.  (Although I have just noticed that this seems a bit buggy, with the displayed assignemnts sometimes being stale....)

Does that make sense?

Reply:
Just checked, there's a bug in (#82985) - basically it will assign attributes based on the attribute group that was selected when the "Track Ball Assign" menu was opened, irrespective of if you change attribute group once you're in that menu. Might be a helpful thing to point out for anyone trying to use it until we fix that bug (I don't know what our policy is on documenting bugs in the manual....?!)

Reply:
I will describe it to say that you need to set the attribute group before pressing Assign.